[2025-07-19T16:14:29.629+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: restaurant_pipeline.stream_to_bronze scheduled__2025-07-19T16:12:00+00:00 [queued]>
[2025-07-19T16:14:29.633+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: restaurant_pipeline.stream_to_bronze scheduled__2025-07-19T16:12:00+00:00 [queued]>
[2025-07-19T16:14:29.633+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-07-19T16:14:29.638+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): stream_to_bronze> on 2025-07-19 16:12:00+00:00
[2025-07-19T16:14:29.640+0000] {standard_task_runner.py:57} INFO - Started process 551 to run task
[2025-07-19T16:14:29.642+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'restaurant_pipeline', 'stream_to_bronze', 'scheduled__2025-07-19T16:12:00+00:00', '--job-id', '833', '--raw', '--subdir', 'DAGS_FOLDER/restaurant_pipeline.py', '--cfg-path', '/tmp/tmpsatvvnm2']
[2025-07-19T16:14:29.643+0000] {standard_task_runner.py:85} INFO - Job 833: Subtask stream_to_bronze
[2025-07-19T16:14:29.665+0000] {task_command.py:416} INFO - Running <TaskInstance: restaurant_pipeline.stream_to_bronze scheduled__2025-07-19T16:12:00+00:00 [running]> on host 28e8e04e957d
[2025-07-19T16:14:29.701+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='moran' AIRFLOW_CTX_DAG_ID='restaurant_pipeline' AIRFLOW_CTX_TASK_ID='stream_to_bronze' AIRFLOW_CTX_EXECUTION_DATE='2025-07-19T16:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-19T16:12:00+00:00'
[2025-07-19T16:14:29.701+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-19T16:14:29.702+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "docker exec -e AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-19T16:12:00+00:00' spark-iceberg spark-submit /home/iceberg/spark/stream_to_bronze.py"]
[2025-07-19T16:14:29.705+0000] {subprocess.py:86} INFO - Output:
[2025-07-19T16:14:31.128+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkContext: Running Spark version 3.5.6
[2025-07-19T16:14:31.129+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkContext: OS info Linux, 6.10.14-linuxkit, aarch64
[2025-07-19T16:14:31.129+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkContext: Java version 17.0.15
[2025-07-19T16:14:31.142+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceUtils: ==============================================================
[2025-07-19T16:14:31.142+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-07-19T16:14:31.142+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceUtils: ==============================================================
[2025-07-19T16:14:31.142+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkContext: Submitted application: StreamToBronze
[2025-07-19T16:14:31.151+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-07-19T16:14:31.156+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceProfile: Limiting resource is cpu
[2025-07-19T16:14:31.156+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-07-19T16:14:31.177+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SecurityManager: Changing view acls to: root,spark
[2025-07-19T16:14:31.177+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SecurityManager: Changing modify acls to: root,spark
[2025-07-19T16:14:31.177+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SecurityManager: Changing view acls groups to:
[2025-07-19T16:14:31.177+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SecurityManager: Changing modify acls groups to:
[2025-07-19T16:14:31.178+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: root, spark; groups with view permissions: EMPTY; users with modify permissions: root, spark; groups with modify permissions: EMPTY
[2025-07-19T16:14:31.203+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-07-19T16:14:31.323+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Utils: Successfully started service 'sparkDriver' on port 46713.
[2025-07-19T16:14:31.336+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkEnv: Registering MapOutputTracker
[2025-07-19T16:14:31.353+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkEnv: Registering BlockManagerMaster
[2025-07-19T16:14:31.366+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-07-19T16:14:31.367+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-07-19T16:14:31.367+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-07-19T16:14:31.378+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-36facd2d-7111-4d08-ba0d-3f341294d3ff
[2025-07-19T16:14:31.384+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-07-19T16:14:31.393+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-07-19T16:14:31.445+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2025-07-19T16:14:31.473+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-07-19T16:14:31.526+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Executor: Starting executor ID driver on host ed7b546796fe
[2025-07-19T16:14:31.526+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Executor: OS info Linux, 6.10.14-linuxkit, aarch64
[2025-07-19T16:14:31.527+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Executor: Java version 17.0.15
[2025-07-19T16:14:31.530+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2025-07-19T16:14:31.531+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@9e587e7 for default.
[2025-07-19T16:14:31.540+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 46489.
[2025-07-19T16:14:31.540+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO NettyBlockTransferService: Server created on ed7b546796fe:46489
[2025-07-19T16:14:31.541+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-07-19T16:14:31.549+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, ed7b546796fe, 46489, None)
[2025-07-19T16:14:31.549+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManagerMasterEndpoint: Registering block manager ed7b546796fe:46489 with 434.4 MiB RAM, BlockManagerId(driver, ed7b546796fe, 46489, None)
[2025-07-19T16:14:31.553+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, ed7b546796fe, 46489, None)
[2025-07-19T16:14:31.553+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, ed7b546796fe, 46489, None)
[2025-07-19T16:14:31.761+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2025-07-19T16:14:31.764+0000] {subprocess.py:93} INFO - 25/07/19 16:14:31 INFO SharedState: Warehouse path is 'file:/app/spark-warehouse'.
[2025-07-19T16:14:32.674+0000] {subprocess.py:93} INFO - 25/07/19 16:14:32 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2025-07-19T16:14:32.680+0000] {subprocess.py:93} INFO - 25/07/19 16:14:32 INFO MetricsSystemImpl: Scheduled Metric snapshot period at 10 second(s).
[2025-07-19T16:14:32.680+0000] {subprocess.py:93} INFO - 25/07/19 16:14:32 INFO MetricsSystemImpl: s3a-file-system metrics system started
[2025-07-19T16:29:21.956+0000] {subprocess.py:93} INFO - 25/07/19 16:29:21 WARN HadoopTableOperations: Error trying to recover version-hint.txt data for s3a://warehouse/bronze/Reservations_raw/metadata/version-hint.text
[2025-07-19T16:29:21.958+0000] {subprocess.py:93} INFO - org.apache.hadoop.fs.s3a.AWSClientIOException: getFileStatus on s3a://warehouse/bronze/Reservations_raw/metadata/version-hint.text: com.amazonaws.SdkClientException: Unable to execute HTTP request: minio: Name or service not known: Unable to execute HTTP request: minio: Name or service not known
[2025-07-19T16:29:21.958+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:214)
[2025-07-19T16:29:21.959+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:175)
[2025-07-19T16:29:21.959+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3799)
[2025-07-19T16:29:21.959+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2025-07-19T16:29:21.959+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.extractOrFetchSimpleFileStatus(S3AFileSystem.java:5401)
[2025-07-19T16:29:21.960+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.open(S3AFileSystem.java:1465)
[2025-07-19T16:29:21.960+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.open(S3AFileSystem.java:1441)
[2025-07-19T16:29:21.960+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.FileSystem.open(FileSystem.java:976)
[2025-07-19T16:29:21.960+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.findVersion(HadoopTableOperations.java:318)
[2025-07-19T16:29:21.961+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.refresh(HadoopTableOperations.java:104)
[2025-07-19T16:29:21.961+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.current(HadoopTableOperations.java:84)
[2025-07-19T16:29:21.961+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.BaseMetastoreCatalog.loadTable(BaseMetastoreCatalog.java:47)
[2025-07-19T16:29:21.961+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.lambda$doComputeIfAbsent$14(BoundedLocalCache.java:2406)
[2025-07-19T16:29:21.962+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ConcurrentHashMap.compute(Unknown Source)
[2025-07-19T16:29:21.962+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.doComputeIfAbsent(BoundedLocalCache.java:2404)
[2025-07-19T16:29:21.962+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.computeIfAbsent(BoundedLocalCache.java:2387)
[2025-07-19T16:29:21.962+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.LocalCache.computeIfAbsent(LocalCache.java:108)
[2025-07-19T16:29:21.962+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.LocalManualCache.get(LocalManualCache.java:62)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.CachingCatalog.loadTable(CachingCatalog.java:166)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.SparkCatalog.load(SparkCatalog.java:643)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.SparkCatalog.loadTable(SparkCatalog.java:159)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.source.IcebergSource.getTable(IcebergSource.java:107)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.source.IcebergSource.inferPartitioning(IcebergSource.java:90)
[2025-07-19T16:29:21.963+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.v2.DataSourceV2Utils$.getTableFromProvider(DataSourceV2Utils.scala:82)
[2025-07-19T16:29:21.964+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.streaming.DataStreamWriter.startInternal(DataStreamWriter.scala:399)
[2025-07-19T16:29:21.964+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.streaming.DataStreamWriter.start(DataStreamWriter.scala:251)
[2025-07-19T16:29:21.964+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2025-07-19T16:29:21.964+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:29:21.964+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2025-07-19T16:29:21.965+0000] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Unknown Source)
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - Caused by: com.amazonaws.SdkClientException: Unable to execute HTTP request: minio: Name or service not known
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleRetryableException(AmazonHttpClient.java:1219)
[2025-07-19T16:29:21.966+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1165)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)
[2025-07-19T16:29:21.967+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)
[2025-07-19T16:29:21.968+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)
[2025-07-19T16:29:21.968+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)
[2025-07-19T16:29:21.968+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)
[2025-07-19T16:29:21.968+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getObjectMetadata(AmazonS3Client.java:1372)
[2025-07-19T16:29:21.968+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$getObjectMetadata$10(S3AFileSystem.java:2545)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:414)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:377)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2533)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2513)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3776)
[2025-07-19T16:29:21.969+0000] {subprocess.py:93} INFO - 	... 35 more
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - Caused by: java.net.UnknownHostException: minio: Name or service not known
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - 	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
[2025-07-19T16:29:21.970+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at com.amazonaws.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:27)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.DelegatingDnsResolver.resolve(DelegatingDnsResolver.java:38)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
[2025-07-19T16:29:21.971+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:376)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at jdk.internal.reflect.GeneratedMethodAccessor21.invoke(Unknown Source)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.conn.ClientConnectionManagerFactory$Handler.invoke(ClientConnectionManagerFactory.java:76)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.conn.$Proxy33.connect(Unknown Source)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
[2025-07-19T16:29:21.972+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.apache.client.impl.SdkHttpClient.execute(SdkHttpClient.java:72)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1346)
[2025-07-19T16:29:21.973+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)
[2025-07-19T16:29:21.974+0000] {subprocess.py:93} INFO - 	... 51 more
[2025-07-19T16:31:06.434+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-07-19T16:31:06.434+0000] {subprocess.py:93} INFO -   File "/home/iceberg/spark/stream_to_bronze.py", line 93, in <module>
[2025-07-19T16:31:06.434+0000] {subprocess.py:93} INFO -     read_kafka(
[2025-07-19T16:31:06.435+0000] {subprocess.py:93} INFO -   File "/home/iceberg/spark/stream_to_bronze.py", line 89, in read_kafka
[2025-07-19T16:31:06.435+0000] {subprocess.py:93} INFO -     .start()
[2025-07-19T16:31:06.436+0000] {subprocess.py:93} INFO -      ^^^^^^^
[2025-07-19T16:31:06.436+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/sql/streaming/readwriter.py", line 1527, in start
[2025-07-19T16:31:06.436+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2025-07-19T16:31:06.436+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2025-07-19T16:31:06.436+0000] {subprocess.py:93} INFO -   File "/opt/bitnami/spark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2025-07-19T16:31:06.437+0000] {subprocess.py:93} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o67.start.
[2025-07-19T16:31:06.437+0000] {subprocess.py:93} INFO - : org.apache.iceberg.exceptions.RuntimeIOException: Failed to refresh the table
[2025-07-19T16:31:06.437+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.refresh(HadoopTableOperations.java:126)
[2025-07-19T16:31:06.437+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.current(HadoopTableOperations.java:84)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.BaseMetastoreCatalog.loadTable(BaseMetastoreCatalog.java:47)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.lambda$doComputeIfAbsent$14(BoundedLocalCache.java:2406)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ConcurrentHashMap.compute(Unknown Source)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.doComputeIfAbsent(BoundedLocalCache.java:2404)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.BoundedLocalCache.computeIfAbsent(BoundedLocalCache.java:2387)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.LocalCache.computeIfAbsent(LocalCache.java:108)
[2025-07-19T16:31:06.438+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.shaded.com.github.benmanes.caffeine.cache.LocalManualCache.get(LocalManualCache.java:62)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.CachingCatalog.loadTable(CachingCatalog.java:166)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.SparkCatalog.load(SparkCatalog.java:643)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.SparkCatalog.loadTable(SparkCatalog.java:159)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.source.IcebergSource.getTable(IcebergSource.java:107)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.spark.source.IcebergSource.inferPartitioning(IcebergSource.java:90)
[2025-07-19T16:31:06.439+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.v2.DataSourceV2Utils$.getTableFromProvider(DataSourceV2Utils.scala:82)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.streaming.DataStreamWriter.startInternal(DataStreamWriter.scala:399)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.streaming.DataStreamWriter.start(DataStreamWriter.scala:251)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[2025-07-19T16:31:06.440+0000] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Unknown Source)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - Caused by: org.apache.hadoop.fs.s3a.AWSClientIOException: getFileStatus on s3a://warehouse/bronze/Reservations_raw/metadata/v0.metadata.json: com.amazonaws.SdkClientException: Unable to execute HTTP request: minio: Name or service not known: Unable to execute HTTP request: minio: Name or service not known
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:214)
[2025-07-19T16:31:06.441+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:175)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3799)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3688)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$exists$34(S3AFileSystem.java:4703)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.trackDuration(IOStatisticsBinding.java:444)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2337)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.trackDurationAndSpan(S3AFileSystem.java:2356)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.exists(S3AFileSystem.java:4701)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.getMetadataFile(HadoopTableOperations.java:241)
[2025-07-19T16:31:06.442+0000] {subprocess.py:93} INFO - 	at org.apache.iceberg.hadoop.HadoopTableOperations.refresh(HadoopTableOperations.java:106)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	... 28 more
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - Caused by: com.amazonaws.SdkClientException: Unable to execute HTTP request: minio: Name or service not known
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleRetryableException(AmazonHttpClient.java:1219)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1165)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)
[2025-07-19T16:31:06.443+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getObjectMetadata(AmazonS3Client.java:1372)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$getObjectMetadata$10(S3AFileSystem.java:2545)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:414)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:377)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2533)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.getObjectMetadata(S3AFileSystem.java:2513)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3776)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	... 37 more
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - Caused by: java.net.UnknownHostException: minio: Name or service not known
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(Unknown Source)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAddressesFromNameService(Unknown Source)
[2025-07-19T16:31:06.444+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress$NameServiceAddresses.get(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName0(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at java.base/java.net.InetAddress.getAllByName(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:27)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.DelegatingDnsResolver.resolve(DelegatingDnsResolver.java:38)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:376)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at jdk.internal.reflect.GeneratedMethodAccessor21.invoke(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.conn.ClientConnectionManagerFactory$Handler.invoke(ClientConnectionManagerFactory.java:76)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.conn.$Proxy33.connect(Unknown Source)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
[2025-07-19T16:31:06.445+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	at com.amazonaws.thirdparty.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.apache.client.impl.SdkHttpClient.execute(SdkHttpClient.java:72)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1346)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 	... 53 more
[2025-07-19T16:31:06.446+0000] {subprocess.py:93} INFO - 
[2025-07-19T16:31:06.468+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO SparkContext: Invoking stop() from shutdown hook
[2025-07-19T16:31:06.468+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2025-07-19T16:31:06.475+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO SparkUI: Stopped Spark web UI at http://ed7b546796fe:4040
[2025-07-19T16:31:06.481+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-07-19T16:31:06.489+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO MemoryStore: MemoryStore cleared
[2025-07-19T16:31:06.489+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO BlockManager: BlockManager stopped
[2025-07-19T16:31:06.491+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-07-19T16:31:06.493+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-07-19T16:31:06.498+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO SparkContext: Successfully stopped SparkContext
[2025-07-19T16:31:06.498+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO ShutdownHookManager: Shutdown hook called
[2025-07-19T16:31:06.499+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO ShutdownHookManager: Deleting directory /tmp/spark-4b2143c0-3b3e-4b31-87c3-26b24ec89f70
[2025-07-19T16:31:06.500+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO ShutdownHookManager: Deleting directory /tmp/spark-08282af8-21d1-4119-aba8-82a0f642a962/pyspark-c1b98a02-0810-4d76-914d-19c26712b115
[2025-07-19T16:31:06.501+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO ShutdownHookManager: Deleting directory /tmp/spark-08282af8-21d1-4119-aba8-82a0f642a962
[2025-07-19T16:31:06.503+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO MetricsSystemImpl: Stopping s3a-file-system metrics system...
[2025-07-19T16:31:06.503+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO MetricsSystemImpl: s3a-file-system metrics system stopped.
[2025-07-19T16:31:06.503+0000] {subprocess.py:93} INFO - 25/07/19 16:31:06 INFO MetricsSystemImpl: s3a-file-system metrics system shutdown complete.
[2025-07-19T16:31:06.526+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-07-19T16:31:06.536+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-07-19T16:31:06.539+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=restaurant_pipeline, task_id=stream_to_bronze, execution_date=20250719T161200, start_date=20250719T161429, end_date=20250719T163106
[2025-07-19T16:31:06.549+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 833 for task stream_to_bronze (Bash command failed. The command returned a non-zero exit code 1.; 551)
[2025-07-19T16:31:06.586+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-07-19T16:31:06.612+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
